src/frameworks/nuxt/index.ts

```diff
@@ -1,33 +1,39 @@
 import { copy, pathExists } from "fs-extra";
 import { readFile } from "fs/promises";
-import { basename, join } from "path";
+import { join } from "path";
 import { gte } from "semver";
-import { BuildResult, findDependency, FrameworkType, relativeRequire, SupportLevel } from "..";
+import { findDependency, FrameworkType, relativeRequire, SupportLevel } from "..";
 
-export const name = "Nuxt";
+export const name = "Nuxt 3";
 export const support = SupportLevel.Experimental;
 export const type = FrameworkType.Toolchain;
```

Comment:
huh, why was this toolchain, should be metaframework

Reply:
Well, [Toolchain](https://github.com/firebase/firebase-tools/blob/4e013b40a3d4069c3758235544d62d12e8c003d3/src/frameworks/index.ts#L89) seems to map to Vite?